Dockerfile

```diff
@@ -1,15 +1,13 @@
-FROM golang:1.7-alpine
+FROM discoenv/golang-base:master
 
-RUN apk update && apk add git
-
-RUN go get github.com/jstemmer/go-junit-report
+ENV CONF_TEMPLATE=/go/src/github.com/cyverse-de/jex-adapter/jobservices.yml.tmpl
+ENV CONF_FILENAME=jobservices.yml
+ENV PROGRAM=jex-adapter
 
 COPY . /go/src/github.com/cyverse-de/jex-adapter
 RUN go install github.com/cyverse-de/jex-adapter
 
 EXPOSE 60000
-ENTRYPOINT ["jex-adapter"]
-CMD ["--help"]
```

Comment:
I've put the new entrypoint (`run-service`) and the default `CMD` into `golang-base`, since they're usually going to be the same.